cdap-app-fabric/src/test/java/io/cdap/cdap/internal/capability/CapabilityManagementServiceTest.java

```diff
@@ -207,6 +210,7 @@ public void testCapabilityRefresh() throws Exception {
     CapabilityConfig disabledConfig = changeConfigStatus(config, CapabilityStatus.DISABLED);
     writeConfigAsFile(externalConfigPath, disabledConfig.getCapability(), disabledConfig);
     capabilityManagementService.runTask();
+    waitState(programId, "STOPPED");
```

Comment:
Here I don't think we need to wait state to get stopped, we need to use other method to check for RUNNING since the program might not exist, at this time, the program should exist